pkg/configuration/configuration.go

```diff
@@ -33,9 +33,12 @@ const (
 	// DefaultToolchainStatusName the default name for the toolchain status resource created during initialization of the operator
 	DefaultToolchainStatusName = "toolchain-status"
 
-	// VarDurationBeforeChangeRequestDeletion specificies the duration before a change request is deleted
+	// VarDurationBeforeChangeRequestDeletion specifies the duration before a change request is deleted
```

Comment:
why is the `VarDurationBeforeChangeRequestDeletion ` variable public?

Reply:
thanks for the review @xcoulon! it's used here: https://github.com/codeready-toolchain/host-operator/blob/master/cmd/manager/main.go#L285